qiita_ware/test/test_ebi.py

```diff
@@ -243,6 +244,27 @@ def test_generate_curl_command(self):
                '%%20ebi_access_key%%3D"') % {'xml_dir': submission.xml_dir}
         self.assertEqual(obs, exp)
 
+    def test_parse_EBI_reply(self):
+        e = EBISubmission(2, 'ADD')
+        curl_result = ""
```

Comment:
`curl_result = ""` is conceptually very different from `curl_result = 'success="true"'` yet the returns are the same. Should the possible states or detail from `parse_EBI_reply` be expanded?